_posts/2021-05-01-monthly-dev-post.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Monthly Dev Post of May 2021
+author: People
+---
+
+Welcome to your Monthly Dev Post of May 2021.
+Every "month" one of these post will be created, to tell you about the latest developments in the world of OpenTTD.
```

Comment:
```suggestion
Every "month," one of these posts will be created, to tell you about the latest developments in the world of OpenTTD.
```

Reply:
Ahem, the comma should stay outside the quoted snippet. It should be `… "month", …` and not `… "month," …`, shouldn't it?

Reply:
From my preliminary research, the placement of a comma before or after a closing quotation mark does not matter in the long run, as long as it is consistent throughout the post.

Reply:
Hmm, I'm not a native English speaker but I would assume it is similar to my mother tongue German. The quotes encapsulates literal speech or cites. Is the comma part of the literal speech or the cite, it has also to be ancapsulated but not when not.

Anyway, as TrueBrain started a new attempt, this has become obsolete.

Reply:
Yeah, sorry about that :D I do appreciate the feedback :)

Regarding this specific case, it is a lovely US vs UK thing. US publishers put it before the close quote, UK publishers after. So both are not wrong. For reference: https://style.mla.org/the-placement-of-a-comma-or-period-after-a-quotation/

In the end, it really doesn't matter, as non of our users will be confused one way or the other. And we are not out to write "strictly correct" text anyway, as the amount of "so" I use ..... my English teacher will hit me for it :P

Anyway, again, I appreciate the feedback. But I don't like the staticness of these dev-posts, so I am looking for another platform that provokes me inter-activeness with our end-users :)